frontend/src/libs/hooks/use-section-update/use-section-update.hook.ts

```diff
@@ -0,0 +1,42 @@
+import {
```

Comment:
you can place this hook in `pages/sites/libs/hooks`
since you will use it not globally but only on that page